api/agents/healer_agent.py

```
@@ -0,0 +1,319 @@
+            content = response.choices[0].message.content
+            self.messages.append({"role": "assistant", "content": content})
+            
+            # Parse response
+            result = parse_response(content)
+            healed_sql = result.get("sql_query", "")
+            
+            # Execute against database
+            error = None
+            try:
+                query_results = execute_sql_func(healed_sql)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)